cc3d/core/PySteppables.py

```diff
@@ -402,6 +402,16 @@ def __init__(self, *args, **kwds):
                                          'targetClusterSurface', 'lambdaClusterSurface', 'type', 'lambdaVecX',
                                          'lambdaVecY', 'lambdaVecZ', 'fluctAmpl']
 
+    @property
+    def shared_steppable_dict(self):
+        """Alias for self.shared_steppable_vars"""
+        return self.shared_steppable_vars
+
+    @property
+    def dict(self):
```

Comment:
I would remove dict function because it might be confusing (too generic). shared_steppable_dict is a better name for the new property. @JulianoGianlupi , ake this change and I will merge the PR